azure/apis/cache/v1alpha2/redis_types.go

```diff
@@ -122,15 +122,21 @@ type SKUSpec struct {
 	Capacity int `json:"capacity"`
 }
 
-// RedisStatus defines the observed state of Redis
+// A RedisSpec defines the desired state of a Redis.
+type RedisSpec struct {
+	runtimev1alpha1.ResourceSpec `json:",inline"`
+	RedisParameters              `json:",inline"`
+}
+
+// A RedisStatus represents the observed state of a Redis.
 type RedisStatus struct {
 	runtimev1alpha1.ResourceStatus `json:",inline"`
 
-	State   string `json:"state,omitempty"`
-	Message string `json:"message,omitempty"`
```

Comment:
Is the `Message` field supposed to be completely removed?

Reply:
Yeah that was intentional - it's never actually set anywhere. I suspect this is the case for a lot of our resources, but this one I checked.